src/modules/back-to-top/back-to-top.component.html

```diff
@@ -0,0 +1,3 @@
+<div #backToTop class="back-to-top" (click)="goToTop()" [ngStyle]="{'background-color': backgroundColor, 'color': color}">
```

Comment:
Also, I don't think having an override for background color and text color is needed, here. We should be prescriptive on our styles. What were your thoughts on this feature?